Bug report for `trillium-proxy`: Compilation fails with --no-default-features

**Describe the bug**
`trillium-proxy` cannot be built without the `upstream-connection-counting` feature enabled.

**To Reproduce**
`cargo check -p trillium-proxy --no-default-features`

```
error[E0432]: unresolved import `trillium_server_common`
 --> proxy/src/forward_proxy_connect.rs:6:5
  |
6 | use trillium_server_common::{Connector, ObjectSafeConnector};
  |     ^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `trillium_server_common`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `trillium-proxy` (lib) due to 1 previous error
```

**Expected behavior**
Successful compilation

--- a/proxy/src/forward_proxy_connect.rs
+++ b/proxy/src/forward_proxy_connect.rs
@@ -2,8 +2,8 @@ use crate::bytes;
 use full_duplex_async_copy::full_duplex_copy;
 use std::fmt::Debug;
 use trillium::{async_trait, Conn, Handler, Upgrade};
+use trillium_client::{Connector, ObjectSafeConnector};
 use trillium_http::{Method, Status};
-use trillium_server_common::{Connector, ObjectSafeConnector};
 use url::Url;
 
 #[derive(Debug)]

